wiki/Gameplay/Hit_object/Hit_circle/en.md

```diff
@@ -6,12 +6,24 @@ tags:
 
 # Hit circle
 
-*Hit circles* are coloured, circular [hit objects](/wiki/Gameplay/Hit_object) present in both [osu!](/wiki/Game_mode/osu!) and [osu!taiko](/wiki/Game_mode/osu!taiko) [beatmaps](/wiki/Beatmap). Their appearance varies between each game mode but will generally require a player to click on/tap the hit circle in time with the beat to earn [score](/wiki/Gameplay/Score) based on how [accurate](/wiki/Gameplay/Accuracy) they were (e.g. if they clicked too early, their points would be reduced). Tapping hit circles gives a very small boost to the [health bar](/wiki/Client/Interface/Health_bar), alongside a bigger boost at the end of a [combo](/wiki/Beatmapping/Combo).
+*Hit circle* are a type of colored, circular [hit object](/wiki/Gameplay/Hit_object) present in [osu!](/wiki/Game_mode/osu!) and [osu!taiko](/wiki/Game_mode/osu!taiko) [beatmaps](/wiki/Beatmap). They are core gameplay elements, requiring players to click on/tap them in time with the music to [score](/wiki/Gameplay/Score) point. The more [accurate](/wiki/Gameplay/Accuracy) is the player the more points he scores, and it provides a very small boost to the [health bar](/wiki/Gameplay/Health) as well as a larger boost at the end of a [combo](/wiki/Beatmapping/Combo).
```

Comment:
we use british english
some of these words are incorrectly put into singular forms <.<

```suggestion
*Hit circles* are a type of coloured, circular [hit objects](/wiki/Gameplay/Hit_object) present in [osu!](/wiki/Game_mode/osu!) and [osu!taiko](/wiki/Game_mode/osu!taiko) [beatmaps](/wiki/Beatmap). They are core gameplay elements, requiring players to click on/tap them in time with the music to [score](/wiki/Gameplay/Score) points. Clicking hit circles more [accurately](/wiki/Gameplay/Accuracy) awards more score, as well as providing a very small boost to the [health bar](/wiki/Gameplay/Health) and a larger one at the end of a [combo](/wiki/Beatmapping/Combo).
```